x-pack/plugins/index_lifecycle_management/server/routes/api/policies/register_create_route.ts

```diff
@@ -11,12 +11,8 @@ import { ElasticsearchClient } from 'kibana/server';
 import { RouteDependencies } from '../../../types';
 import { addBasePath } from '../../../services';
 
-async function createPolicy(client: ElasticsearchClient, name: string, phases: any): Promise<any> {
-  const body = {
-    policy: {
-      phases,
-    },
-  };
+async function createPolicy(client: ElasticsearchClient, name: string, policy: any): Promise<any> {
```

Comment:
On L50 we have `typeof bodySchema.type`. We should be able to extract a type from it and use it here instead of `any`. WDYT?